.github/workflows/ci.yml

```diff
@@ -23,12 +23,32 @@ jobs:
             ${{ runner.os }}-maven-
```

Comment:
We discussed cutting out all the non-LTS versions of Java in the matrix above, we still doing that?

Reply:
We are using JDK 19 to test Java 8 - 19 version, we still need to keep the matrix for the build